app/api/schema/event_statistics.py

```diff
@@ -76,9 +76,18 @@ def sessions_canceled_count(self, obj):
         ).count()
 
     def speakers_count_type(self, obj, state='pending'):
-        return SessionsSpeakersLink.query.filter_by(
-            event_id=obj.id, session_state=state, deleted_at=None
-        ).count()
+        return (
+            Speaker.query.join(
+                SessionsSpeakersLink, SessionsSpeakersLink.speaker_id == Speaker.id
+            )
+            .distinct(Speaker.id)
+            .filter(
+                Speaker.event_id == obj.id,
+                Speaker.deleted_at == None,
+                SessionsSpeakersLink.session_state == state,
```

Comment:
Join session and use Session.state instead

Reply:
@iamareebjamal There is no column in Session for joining them should I add that in the schema? :sweat_smile:

Reply:
SessionSpeakerLink has session ID as well

Reply:
Ohh so you want me to join session in addition to session speaker link? what additional filtering do we need in that case @iamareebjamal

Reply:
I am getting a bit confused if we only link Session and SessionSpeakerLink we will lose our ability to filter out deleted speakers :sweat_smile:

Reply:
Please join the technical meeting at 7 PM and we'll discuss there. What is your email?

Reply:
itm2017004@iiita.ac.in @iamareebjamal